framework/CHANGELOG.md

```diff
@@ -52,7 +52,8 @@ Yii Framework 2 Change Log
 - Enh #11187: migrate command now generates phpdoc for table migrations (Faryshta)
 - Enh #11254: Added ability to attach RBAC rule using class name (mdmunir)
 - Enh: Added `StringHelper::countWords()` that given a string returns number of words in it (samdark)
-- Enh #11207: migrate command can create foreign keys. (faryshta)
+- Enh #11207: migrate command can create foreign keys. (Faryshta)
```

Comment:
this was a minimal change, just one letter from lower case to uppercase, it can be undone without issue.